Dynavity/Dynavity/view/CanvasSelectionView.swift

```diff
@@ -1,8 +1,14 @@
 import SwiftUI
 
 struct CanvasSelectionView: View {
-    @Binding var canvases: [String]
+    struct CanvasDetail: Hashable {
```

Comment:
Just checking, this is meant to be a temporary substitute for the actual canvas model right? If so, the canvas model wouldn't store `isSelected` I think. Is it possible to save `isSelected` as a state of `CanvasThumbnailView` instead? I think we can just add a `TODO` note for now and we can implement this together with the actual canvas model some other time

Reply:
Yep! I'll add the `TODO` note. As for having `isSelected` as the state of `CanvasThumbnailView`, that would mean that the `CanvasSelectionView` would have to store an array of `CanvasThumbnailView`s instead? I don't think that'll work out.

I think you're right that the `isSelected` shouldn't be in the canvas model. As for that I'll think another way of getting the same behavior once we implement this with the actual canvas model!